lib/header.ml

```diff
@@ -16,16 +16,30 @@
  *
   }}}*)
 
+exception Found_int of int
+
+let compare_caseless x y =
+  if String.(length x <> length y)
+  then String.compare x y
```

Comment:
Agreed. I should probably just get rid of that early check since it's just a premature optimization.